CI passes on the last commit, so the mechanism must be working. 

There is issue #23049, which has this list of failures:
```
2023-01-19T14:03:09.5394521Z FAILED numpy/core/tests/test_umath.py::TestComplexFunctions::test_precisions_consistent - AssertionError: 
2023-01-19T14:03:09.5394788Z Arrays are not almost equal to 15 decimals fch-fcl <ufunc 'arctan'>
2023-01-19T14:03:09.5394889Z  ACTUAL: (1.0172219276428222656+0.40235948562622070312j)
2023-01-19T14:03:09.5394991Z  DESIRED: (1.0172219678978514+0.40235947810852507j)
2023-01-19T14:03:09.5395442Z FAILED numpy/core/tests/test_umath.py::TestComplexFunctions::test_branch_cuts - AssertionError: (array([-1.57079633-0.54930614j, -1.57079633+0.54930614j]), array([1.57079633-0.54930614j, 1.57079633+0.54930614j]))
2023-01-19T14:03:09.5395957Z FAILED numpy/core/tests/test_umath.py::TestComplexFunctions::test_branch_cuts_complex64 - AssertionError: (array([-1.5707963-0.54930615j, -1.5707963+0.54930615j], dtype=complex64), array([1.5707884-0.54930615j, 1.5707884+0.54930615j], dtype=complex64))
2023-01-19T14:03:09.5396333Z FAILED numpy/core/tests/test_umath.py::TestComplexFunctions::test_loss_of_precision[complex64] - RuntimeWarning: divide by zero encountered in divide
2023-01-19T14:03:09.5396706Z FAILED numpy/core/tests/test_umath.py::TestComplexFunctions::test_loss_of_precision[complex128] - RuntimeWarning: divide by zero encountered in divide
2023-01-19T14:03:09.5397075Z FAILED numpy/core/tests/test_umath.py::TestComplexFunctions::test_loss_of_precision[complex256] - RuntimeWarning: divide by zero encountered in divide
```

I unskipped them, tests passed locally, so I will mark this as fixes #23049

--- a/numpy/_core/tests/test_umath.py
+++ b/numpy/_core/tests/test_umath.py
@@ -17,7 +17,8 @@ from numpy.testing import (
     assert_, assert_equal, assert_raises, assert_raises_regex,
     assert_array_equal, assert_almost_equal, assert_array_almost_equal,
     assert_array_max_ulp, assert_allclose, assert_no_warnings, suppress_warnings,
-    _gen_alignment_data, assert_array_almost_equal_nulp, IS_WASM, IS_PYPY
+    _gen_alignment_data, assert_array_almost_equal_nulp, IS_WASM, IS_MUSL, 
+    IS_PYPY
     )
 from numpy.testing._private.utils import _glibc_older_than
 
@@ -1888,6 +1889,7 @@ class TestFRExp:
     @pytest.mark.parametrize("dtype", ['f', 'd'])
     @pytest.mark.skipif(not sys.platform.startswith('linux'),
                         reason="np.frexp gives different answers for NAN/INF on windows and linux")
+    @pytest.mark.xfail(IS_MUSL, reason="gh23049")
     def test_frexp(self, dtype, stride):
         arr = np.array([np.nan, np.nan, np.inf, -np.inf, 0.0, -0.0, 1.0, -1.0], dtype=dtype)
         mant_true = np.array([np.nan, np.nan, np.inf, -np.inf, 0.0, -0.0, 0.5, -0.5], dtype=dtype)

